custom_components/miio2/manifest.json

```diff
@@ -3,12 +3,13 @@
   "name": "Xiaomi miio vacuum STYJ02YM",
   "version": "2021.8.0",
   "documentation": "https://github.com/KrzysztofHajdamowicz/home-assistant-vacuum-styj02ym",
+  "issue_tracker": "https://github.com/KrzysztofHajdamowicz/home-assistant-vacuum-styj02ym/issues",
   "requirements": [
     "construct==2.10.56",
-    "python-miio==0.5.9.1"
+    "python-miio==0.5.9.2"
   ],
   "dependencies": [],
   "codeowners": ["@nqkdev", "@KrzysztofHajdamowicz"],
   "config_flow": false,
-  "iot_class": "local_polling"
+  "iot_class": "cloud_polling"
```

Comment:
Why `cloud_polling`?
According to [Documentation](https://developers.home-assistant.io/docs/creating_integration_manifest/#iot-class) `local_polling` is acceptable value and this integration communicates with vacuum locally.

Reply:
How does it communicate locally? I thouhght that it sends commands to the xiaomi server and the only way to communicate locally is to root the vacuum, am I wrong?

Reply:
https://github.com/KrzysztofHajdamowicz/home-assistant-vacuum-styj02ym/blob/0bba5b0b5d1115201ee4ff3759c8fa76482ab53b/custom_components/miio2/vacuum.py#L197

It uses IP and token of vacuum to connect locally using Xiaomi protocol.